nix/index.tt

```diff
@@ -21,16 +21,31 @@
 
   [% WRAPPER makePopover title="<i class='fa fa-cloud-download'></i> Get Nix" %]
 
-    <p>On Linux and Mac OS X, the <strong>easiest way to install
+    <p>On Linux and macOS, the <strong>easiest way to install
     Nix</strong> is to run the following shell command (as a user
     other than <tt>root</tt>):</p>
 
-    <pre><span class="muted">$</span> curl https://nixos.org/nix/install | sh</pre>
+    <pre>
+<span class="command">curl -o install-nix-[%latestNixVersion%] https://nixos.org/nix/install</span>
+<span class="command">echo "[%nixInstallHashsha256%]  install-nix-[%latestNixVersion%]" | (shasum -p 256 || sha256sum -c) 2> /dev/null</span>
+<span class="command">sh install-nix-[%latestNixVersion%]</span>
+</pre>
 
     <p>This script will download a distribution-independent binary
     tarball containing Nix and its dependencies, and unpack it in
     <tt>/nix</tt>.</p>
 
+    <p>If you would like to verify the installer with GPG, import the
+    <a href="[%root%]/edolstra.gpg">signing key</a> (also on
+    <a href="https://github.com/NixOS/nixos-homepage/blob/master/edolstra.gpg">GitHub</a>)
+    and verify with the signature:</a>
+
+    <pre>
+<span class="command">gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE</span>
+<span class="command">curl -o install-nix-[%latestNixVersion%].sig https://nixos.org/nix/install.sig</span>
+<span class="command">gpg2 --verify ./install-nix-[%latestNixVersion%].sig</span>
```

Comment:
To echo the comment above, this command requires/assumes there is a file signed with this signature by the name of `install-nix-[%latestNixVerison%]` (e.g. "same filename, without the `.sig`"). Not sure if it's necessary to note, based on the population that would verify via gpg - but something to keep in mind =]